krakenapi.go

```diff
@@ -340,6 +342,31 @@ func (api *KrakenAPI) OpenOrders(args map[string]string) (*OpenOrdersResponse, e
 	return resp.(*OpenOrdersResponse), nil
 }
 
+// OpenPositions returns all open orders
+func (api *KrakenAPI) OpenPositions(args map[string]string) (*OpenPositionsResponse, error) {
```

Comment:
Would it make sense to make the arguments more concrete ?
For example docals could be a bool. This would make the usage easier.

Reply:
Ideally probably use a custom struct to define what can go into this, admittedly I was just following the format of how the rest of the SDK is currently designed.

A custom struct would go a long way toward allowing custom options and future expandability.